Lib/asyncio/tasks.py

```diff
@@ -639,10 +639,34 @@ def __sleep0():
     """
     yield
 
+def __check_delay(delay):
```

Comment:
Nitpick: Does it really need double underscore? If you want to mark something as "private API" use one leading underscore